catalog/app/utils/gtm.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+
+import cfg from 'constants/config'
+
+function loadScript(src: string) {
+  const s = window.document.createElement('script')
+  s.type = 'text/javascript'
+  s.async = true
+  s.src = src
+
+  const x = window.document.getElementsByTagName('script')[0]
+  x.parentNode?.insertBefore(s, x)
```

Comment:
this would fail silently if `x.parentNode` is null-ish (which shouldn't happen, but...), so i'd rather add `assert x.parentNode` before this call to fail loudly and make TS happy

Reply:
Ok, I added `invariant`